src/coreclr/nativeaot/System.Private.CoreLib/src/System/Reflection/ConstructorInvoker.cs

```diff
@@ -67,8 +67,8 @@ public object Invoke(object? arg1, object? arg2)
             }
 
             StackAllocatedArguments argStorage = default;
-            argStorage._args.Set(0, arg1);
```

Comment:
Does the JIT generate exact same or better code for this as before this change?

Reply:
In my initial testing, yes, it was the same. It's certainly possible with more complicated uses it's not. Let's see what the bot comes back with.

Reply:
The bot agrees.